lib/std/zig/system/macos.zig

```diff
@@ -452,3 +454,25 @@ test "version_from_build" {
         std.testing.expect(std.mem.eql(u8, sver, pair[1]));
     }
 }
+
+/// Detect SDK path on Darwin.
+/// Calls `xcrun --show-sdk-path` which result can be used to specify
+/// `-syslibroot` param of the linker.
+/// The caller needs to free the resulting path slice.
+pub fn getSDKPath(allocator: *mem.Allocator) ![]u8 {
+    assert(std.Target.current.isDarwin());
+    const argv = &[_][]const u8{ "/usr/bin/xcrun", "--show-sdk-path" };
```

Comment:
Is this intentionally ignoring PATH with respect to `xcrun`? (not ignoring PATH would look like chopping off the "/usr/bin/")